gpii/node_modules/flowManager/src/FlowManager.js

```diff
@@ -318,13 +332,10 @@ gpii.flowManager.local.resetAtStart = function (lifecycleManager, resetAtStart,
     defaultSettingsDataPromise.then(function (defaultSettingsData) {
         var defaultSnapshot = defaultSettingsData.defaultSnapshot;
         if (resetAtStart && defaultSnapshot) {
-            fluid.log("Resetting on system startup to default settings: ", defaultSnapshot);
```

Comment:
These removed logging lines at 321, 324, 327 are useful to report if the reset at start is performed and the result. Could you not remove them? Thanks.

Reply:
Oops.  I had repurposed them for my own "TRACE:" debugging messages, and when I search and removed those, these matched.  Thanks for catching that.